sdks/python/run_pylint.sh

```diff
@@ -118,4 +99,4 @@ if [ -n "${TESTS_MISSING_MAIN}" ]; then
   done
   echo
   exit 1
-fi
+fi
```

Comment:
Let's add a new line at the end of file here and in sdks/python/run_pylint_2to3.sh.

Reply:
done